app/models/runs_extractors.rb

```diff
@@ -48,7 +48,8 @@ def extract(classification)
 
     raise Extractor::ExtractionFailed.new('One or more extractors failed') if has_errors
 
-    return if extracts&.compact.blank?
+    extracts = extracts&.select{ |e| e != Extractor::NoData }&.compact
```

Comment:
is this the first of a double filtering of NoDatas, along with the one in the ExtractWorker?

Reply:
yep removed the latter